howso/get_cases.amlg

```diff
@@ -89,6 +89,7 @@
 		; 	}
 		; }
 		(assoc)
+		(call !ValidateParameters)
```

Comment:
Is this related to the PR title or just a fix?

Reply:
completely unrelated, its purpose here is to error out if unexpected parameters (any parameters in this case) are given

Reply:
done